doc/CMakeLists.txt

```diff
@@ -39,6 +39,16 @@ if(NOT LATEX_PDFLATEX_FOUND OR NOT LATEXMK)
   message(WARNING "LaTeX components not found. PDF build will not be available.")
 endif()
 
+execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} -c "import doxmlparser"
+    RESULT_VARIABLE EXIT_CODE
+    OUTPUT_QUIET ERROR_QUIET
+)
+
+if (NOT ${EXIT_CODE} EQUAL 0)
+  message(FATAL_ERROR "'doxmlparser' Python package was not found")
+endif()
+
```

Comment:
not needed, just update requirements

Reply:
But `doxmlparser` is not on PyPi, so it wouldn't be installed and we'd just have the import failure - isn't that worse? Am I missing something?

Reply:
Then IMO this should not be merged as-is, because it's breaking the use case of "build the docs locally after installing requirements from pip"

Reply:
@edersondisouza if `doxmlparser` is not in PyPi then why did you add it to `doc/requirements.txt`?

Reply:
Oh, it is now. I've sent a PR to doxygen to publish it to PyPi (https://github.com/doxygen/doxygen/pull/10328) that eventually got merged and it is on PyPi now (https://pypi.org/project/doxmlparser/).